packages/manager/modules/pci/src/projects/project/storages/databases/database/general-information/general-information.html

```diff
@@ -407,25 +407,6 @@
                         </div>
                     </oui-tile-description>
                 </oui-tile-definition>
-                <oui-tile-definition
-                    data-term="{{:: 'pci_databases_general_information_replicas_label' | translate }}"
```

Comment:
Both translation keys:
- `pci_databases_general_information_replicas_label`
- `pci_databases_general_information_replicas_link`

can be safely removed from following translation file:
- `packages/manager/modules/pci/src/projects/project/storages/databases/database/general-information/translations/Messages_fr_FR.json:`

Reply:
Done: https://github.com/ovh/manager/pull/5583/commits/a8058df864f9d9be07c2b26aa79bcbfe1ae93aee